### ISSUE TITLE
This is great, some pointers to implement it?

### ISSUE THREAD

**Issue by FartyPants** [sam-paech/antislop-sampler#8]
I want to implement it as oobabooga extension as it has logits_processor_modifier external function, which I think is the inflection point where this should be implemented. 

def logits_processor_modifier(processor_list, input_ids):
    """
    Adds logits processors to the list, allowing you to access and modify
    the next token probabilities.
    Only used by loaders that use the transformers library for sampling.
    """
    processor_list.append(MyLogits())
    return processor_list

So now the question is how to wrap the code to the LogitsProcessor class.

**Comment by sam-paech** [comment]
It sounds like you have the ingredients to make it work. One possible issue is that without kvcache management, inference will be slow. This is because you have to check the slop list every token, so you won't benefit from automatic cache management that would otherwise be happening with longer generations. I'm not sure if ooba gives you ways to manage kvcache.

**Comment by FartyPants** [comment]
Actually, since this already uses OpenAi API, it would be mostly redundant, because it should already work in ooba through the openAi API.

**Comment by FartyPants** [comment]
Since this already uses OpenAi API, extension is redundant. It should already work through the openAi API fine.
